c_src/libSigPyProcSpec.cpp

```diff
@@ -0,0 +1,249 @@
+#include <cmath>
+#include <cstring>
+#include <fftw3.h>
+#include <complex.h>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+
+#include <libUtils.hpp>
+
+namespace py = pybind11;
+
```

Comment:
I'm going to suggest comments for all the functions, but that makes me a hypocrite...
`/* Complex-to-complex FFT */`